scripts/init.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+ERR='\033[0;31m[ERROR]\033[0m'
+WRN='\033[1;33m[WARNING]\033[0m'
+INF='\033[1;34m[INFO]\033[0m'
+
+TMP=/tmp
+
```

Comment:
The main reason was for the user to be able to change the location of temporary files. Removed since it is not needed anymore.